changelog/_unreleased/2020-09-15-allow-html-characters-in-product-name.md

```diff
@@ -0,0 +1,9 @@
+---
+title:  Add support for autowiring of entity repositories
```

Comment:
![Oh really?](https://i.giphy.com/media/l0MZ9TvI0tXPeoF9K/giphy.webp)

Reply:
:laughing:

Reply:
Damn, now everyone knows that my changelog:create did not work and I copied it :D